modules/rostests/tests/combotst/combotst.c

```diff
@@ -7,7 +7,7 @@
 * or make this more complex. Also Fonts are not heavily used.
 
 * This source code is in the PUBLIC DOMAIN and has NO WARRANTY.
-* by Waldo Alvarez Ca�izares <wac at ghost.matcom.uh.cu>, started July 11, 2003. */
+ * by Waldo Alvarez Ca�izares <wac at ghost.matcom.uh.cu>, started July 11, 2003. */
```

Comment:
```suggestion
* by Waldo Alvarez Ca�izares <wac at ghost.matcom.uh.cu>, started July 11, 2003. */
```
rervert that change please!